app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewHolderFactory.kt

```diff
@@ -415,8 +415,14 @@ private fun navigateToCreationDetails(view: View, details: TransactionView.Creat
                 statusTextRes = details.statusText,
                 dateTimeText = details.dateTimeText,
                 implementation = details.implementation,
+                implementationName = details.implementationInfo.name,
```

Comment:
Since `implementationInfo` is optional (on the backend side). Should we have a fail safe String to show here? Like "Unknown" for example? Same for the other *Infos.